examples/react-native/__tests__/ts/authentication/manage-email-password-users.test.tsx

```diff
@@ -0,0 +1,75 @@
+// :snippet-start: register-user
+import React from 'react';
+import {useApp, UserProvider, AppProvider} from '@realm/react';
+import Realm from 'realm';
+// :remove-start:
+import {render, fireEvent, waitFor} from '@testing-library/react-native';
+import {View, Button, Text} from 'react-native';
+
+const APP_ID = 'example-testers-kvjdy';
+
+const rand = Date.now();
+const userCreds = {
+  username: `user+email-pw-${rand}@example.com`,
+  password: 'abc123',
+};
+let higherScopedUser: Realm.User;
+// :remove-end:
+
+function AppWrapper() {
+  return (
+    <View>
+      <AppProvider id={APP_ID}>
+        <RegisterUser />
+        <UserProvider fallback={LogIn}>
```

Comment:
```suggestion
        <UserProvider fallback={RegisterUser}>
```
I would set `RegisterUser` as the fallback and remove `LogIn`.